packages/relay/src/lib/eth.ts

```diff
@@ -80,6 +80,7 @@ export class EthImpl implements Eth {
   static blockLatest = 'latest';
   static blockEarliest = 'earliest';
   static blockPending = 'pending';
+  static fewBlocksGreater = 3;
```

Comment:
I was thinking a similar thing on the dynamic property aspect.
Each blocks at top performance is 2 secs so this gives us a 6 second window which is not unreasonable.
So far as the error also shows up in metrics as something we can react to I think we should be good to go with a value of `fewBlocksGreater = 5` to start.
This gives as 10 secs which is the worst E2E for transaction submission to query request path we might support.

Reply:
Updated to 5, but I agree that making it a configurable property does seem to make sense.  I started with 3 because usually few is understood to be about 3.